include/inviwo/core/processors/progressbar.h

```diff
@@ -49,14 +49,16 @@ class IVW_CORE_API ProgressBarObserver : public Observer {
     /**
      * This method will be called when observed object changes.
      * Override it to add behavior.
+     * @param New progress 
```

Comment:
what is the range of the param?

Reply:
Updated documentation of ProgressBar and related classes